yb-voyager/cmd/importData.go

```diff
@@ -140,7 +159,12 @@ func importData(importFileTasks []*ImportFileTask) {
 	}
 
 	targetDBVersion := tdb.GetVersion()
-	fmt.Printf("Target YugabyteDB version: %s\n", targetDBVersion)
+
+	if tconf.TargetDBType == YUGABYTEDB {
+		fmt.Printf("Target YugabyteDB version: %s\n", targetDBVersion)
+	} else {
+		fmt.Printf("%s version: %s\n", tconf.TargetDBType, targetDBVersion)
```

Comment:
This single statement will work correctly for both YB and Oracle. Why do you need to include it in the `if..else`?